src/internal/views/nunjucks/billing/batch-cancel-or-confirm.njk

```diff
@@ -9,11 +9,13 @@
     <div class="govuk-grid-column-two-thirds">
       {{ title(pageTitle, null, null, true) }}
 
-      <span class="govuk-caption-l">Bill run {{  batch.billRunId }}</span>
+      <span class="govuk-caption-l">{{ batch.region.displayName }} bill run {{  batch.billRunId }}</span>
       <h2 class="govuk-heading-l">{{ secondTitle }}</h2>
-      <div class="govuk-grid-row govuk-!-margin-bottom-3">
+
         {{ batchMetadata(batch) }}
-      </div>
+
+        <br />
```

Comment:
I feel like our use of line breaks is a bit naughty. Would it be better to wrap the output from `{{ batchMetadata(batch) }}` in a div and apply a govuk overridden bottom margin?